change/@ni-nimble-tokens-8856d58c-6aa1-4fe1-b008-b10544d13034.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update assembly to .net 6",
+  "packageName": "@ni/nimble-tokens",
+  "email": "rajsite@users.noreply.github.com",
+  "dependentChangeType": "none"
```

Comment:
The only change is to the nuget package, so no need to bump dependent npm packages that depend on tokens (i.e. nimble-components and nimble-angular). This should be a patch release of the nimble-tokens npm package with zero changes but changes to the NimbleTokens nuget package to .NET 6.